src/Symfony/Component/DependencyInjection/ParameterBag/EnvPlaceholderParameterBag.php

```diff
@@ -71,6 +71,12 @@ public function getEnvPlaceholders()
      */
     public function mergeEnvPlaceholders(self $bag)
     {
-        $this->envPlaceholders = array_merge_recursive($this->envPlaceholders, $bag->getEnvPlaceholders());
+        $newPlaceholders = $bag->getEnvPlaceholders();
```

Comment:
Pushed changes, and this time I **actually tried it out** on a project. I think I put too much faith in the tests last time. That or I didn't write enough tests... 
Are there tests for actual retrieving of a parameter after the bag has been merged?